docs/sphinx/reference-sources.rst

```diff
@@ -463,7 +463,15 @@ Source Definition Structure (obs_source_info)
    - **OBS_ICON_TYPE_TEXT**            - Text
    - **OBS_ICON_TYPE_MEDIA**           - Media
    - **OBS_ICON_TYPE_BROWSER**         - Browser
-   - **OBS_ICON_TYPE_CUSTOM**          - Custom (not implemented yet)
+   - **OBS_ICON_TYPE_CUSTOM**          - Custom
+
+.. member:: char *(*obs_source_info.get_dark_icon)(void *type_data)
+
+   Gets icon used for dark themes. Make sure icon_type is set to OBS_ICON_TYPE_CUSTOM.
+
+.. member:: char *(*obs_source_info.get_light_icon)(void *type_data)
+
+   Gets icon used for light themes. Make sure icon_type is set to OBS_ICON_TYPE_CUSTOM.
```

Comment:
This does not make it clear that the returned `char *` should be a file path.

Reply:
Fixed to add that it is a file path.